pom.xml

```diff
@@ -184,6 +184,8 @@
 
         <!-- Forbidden API related properties -->
         <forbiddenapis-junit.path>forbidden-junit.txt</forbiddenapis-junit.path>
+
+        <hibernate-validator-parent.basedir>.</hibernate-validator-parent.basedir>
```

Comment:
I would have expected it to be redefined in each submodule? At least that's what I needed to do for BV TCK. Do you have a clever trick to avoid that?